platform/ui/src/components/SplitButton/SplitButton.tsx

```diff
@@ -183,77 +183,83 @@ const SplitButton = ({
 
   const listItemRenderer = renderer || DefaultListItemRenderer;
 
-  return (
-    <OutsideClickHandler onOutsideClick={outsideClickHandler}>
```

Comment:
If you take a look at the https://www.npmjs.com/package/react-outside-click-handler documentation, it mentions that there is a disabled parameter which is specifically designed to enable/disable this, without mounting/unmounting the entire tree.  The cost of setting disabled is also much lower than that of adding/removing the tree, and that makes the GUI more responsive.  Could you use disabled instead?